sale_stock_return_request/models/stock_return_request.py

```diff
@@ -27,15 +27,14 @@ def _action_confirm(self):
 
     def action_view_sales(self):
         """Display returned sales"""
-        action = self.env.ref("sale.action_orders")
-        result = action.read()[0]
-        result["context"] = {}
+        action = self.env["ir.actions.act_window"]._for_xml_id("sale.action_orders")
+        action["context"] = {}
```

Comment:
Squash this changes with your migration commit

Reply:
I've made that fix in a separate commit, so it could be cherry-picked to v15

Reply:
Yes, but you can simply include it in your migration commit for this one

Reply:
Done!